pkg/util/rollout_utils.go

```diff
@@ -164,3 +164,14 @@ func DumpJSON(o interface{}) string {
 func EncodeHash(data string) string {
 	return fmt.Sprintf("%x", sha256.Sum256([]byte(data)))
 }
+
+func NextBatchIndex(rollout *rolloutv1beta1.Rollout, CurrentStepIndex int32) int32 {
+	if rollout == nil {
+		return 0
+	}
+	allSteps := int32(len(rollout.Spec.Strategy.GetSteps()))
+	if CurrentStepIndex >= allSteps {
+		return 0
```

Comment:
plz comment about the wrapping logic, why the next index of last step is 0

Reply:
done

Reply:
can we set the NextStepIndex of last step to -1, so as to avoid confusion